### ISSUE TITLE
secure authorization and token endpoint for authorization_code flow, RFC7636 and RFC9449

### ISSUE THREAD

**Issue by woutslakhorst** [nuts-foundation/nuts-node#3012]

**Comment by reinkrul** [comment]
Since DPoP breaks the example deployment with APISIX, I added a feature request for it: https://github.com/apache/apisix/issues/11219

**Comment by woutslakhorst** [comment]
DPoP is optional so no problem for examples... Only not usable in production yet?

**Comment by reinkrul** [comment]
Indeed, I didn't mean it was blocking, just added a reference here for keeping.
